app/src/main/java/com/ciandt/dragonfly/example/shared/BaseActivity.kt

```diff
@@ -114,6 +123,17 @@ abstract class BaseActivity(protected var hasDebugDrawer: Boolean = true) : AppC
         return pendingPermissions
     }
 
+    private fun checkLogin() {
```

Comment:
I would rename this method to enforceLoginIfNeeded(). Maybe we will have to add a method, overriden by BaseActivity children that defines whether or not the current activity requires the user to be signed in. That would be useful, for example, for the About and Terms & Conditions activities.